lambdas/question/src/main/java/uk/gov/di/ipv/cri/kbv/api/handler/QuestionHandler.java

```diff
@@ -51,31 +49,26 @@
     private final KBVService kbvService;
     private final AuditService auditService;
     private final ConfigurationService configurationService;
-    private final Clock clock;
 
     @ExcludeFromGeneratedCoverageReport
     public QuestionHandler() {
         this.objectMapper = new ObjectMapper().registerModule(new JavaTimeModule());
-        this.kbvStorageService = new KBVStorageService();
         this.personIdentityService = new PersonIdentityService();
```

Comment:
On the whole this is really good, just some tiny bit, I think we should pass through the configurationService to the PersonIdentityService and AuditService as well, so we should have a constructor that just takes a configurationService in PersonIdentityService / AuditService I suppose this could happen in some subsequent PR

Reply:
Indeed, we can look at this in a subsequent PR